.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,85 @@
+name: Release docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/**"
+      - "logo/**"
+      - "*.md"
+  workflow_dispatch: {}
+      
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.after || github.ref }}
+  cancel-in-progress: true
```

Comment:
If a regular release build is running and then this job is triggered before it completes, we'll likely run into a problem because they could both try to tag the same version. We can't make their groups the same because if this cancels the other full build, it won't include the changes from it. A similar race condition happens if this build is running first and then the full build starts.

I was on board with this approach although I think this issue is a blocker. WDYT?

Reply:
It really is quite an issue. I did consider about the concurrency issue a lot and that's how I settled on `github.event.after` in the id so that every job will have a unique id. But it was an oversight on my end not thinking even if commits are different, release could be the same. Now the one remaining idea I have is to just add a job to check all the changes since last commit with a script, add this workflow as another job in the build and then conditionally trigger the jobs based on file check output, instead of filtering paths on the workflow level. Though it feels like this is all getting a bit complicated so I don't mind scrapping the pr and handing the issue over to someone else with a better idea. Do you think the idea could end up as a proper solution?

Reply:
Yeah I think this approach will have to be scrapped unfortunately, can't really think of a way to overcome this issue when there is a separate workflow.

> Now the one remaining idea I have is to just add a job to check all the file changes since last commit with a script, add this workflow as another job in the build and then conditionally trigger the jobs based on file check output, instead of filtering paths on the workflow level.

I do think the idea would work (have both seen and done similar things) but it may be time to take a step back and evaluate the core pieces here. Notably: The build system (nx), the workflows, and the boundaries between them. It's a little messy right now which makes this PR difficult. Additionally, as Wing matures, versioning will likely change which complicates this more.

So for now we can close this, but I really love the work you've done on the workflows ❤️